### DISCUSSION TITLE
CRC32 not found (N64 and Megadrive ROMS)

### DISCUSSION THREAD

**Discussion by RossieAC** [sanni/cartreader#893] [Need help]
### OSCR firmware version

9.5 to 12.4 - see below

### OSCR hardware version

HW5 Rev3

### Attach OSCR_LOG.txt file here

(N64 example)

OSCR HW5 V12.4

[+] Nintendo 64 (3V)

[+] Game Cartridge

Searching database...

GAMEPAK ERROR

Title: THE LEGEND_OF ZELDA
Serial: NZLP
CRC1: 90449569
Cartridge unknown

Press Button...

[+] 64 MB

[+] Read ROM

Saving to N64/ROM/GPERROR/20/...
[*******************]
CRC32... D1139BBC -> Not found

(Megadrive example)

OSCR HW5 V9.5

[+] Mega Drive

[+] Game Cartridge

Cart Info
 
Name: 088888880808888
bramCheck: 0030
Size: 24 MBit
ChkS: 0038
Sram: None
 

[+] Read Rom

Saving to MD/ROM/088888880808888/42/...
[*******************]
Internal checksum...Error
35FC

CRC32... 2B0BDF38 -> Not found

### System used

N64 Cartridge

### Describe the issue.

None of my many N64 or Megadrive cartridges  will dump ROMs. (I don't have other types so slots not enabled or installed)
Every cart I try on gives result "CRC32 not found" 
Most N64 cartridges report Gamepak error but the cart name is displayed. 
Only Goldeneye does not report Gamepak error, it saves a Z64 file but still reports CRC32 not found and Mupen64 will not load it. 
Have ran self test as web advice says
I have cleaned cart terminals as web advice says
I have checked solder joints as web advice says
I have flashed OCRs 9.5 to 12.4.  Earlier versions drivers don't install, later versions report compiling error "checkAltConf exit status 1 'checkButton' was not declared in this scope" I don't know how to fix that
I have tried disabling clock generator
Now considering new parts and build but don't want to waste money, so I'm wondering if this is a common / known / solved problem?

**Comment by RossieAC** [comment]
PS all N64 dumps save Z64 files which attempt but do not load in emulator. Megadrive dumps save a .BIN file which are not recognised by an emulator

**Comment by PsyK0p4T** [comment]
Hi. Have you successfully dumped any other systems with your OSCR ? Just to isolate issues like SD card corruption etc.

For the N64, if you see the game title displayed well on screen, then it's a good start. Try to compare your z64 roms with current existing roms, like with an hex editor, to see how different they are.
 
For the MD, your cart seems not recognized well at all. The header is already messy ("Name: 088888880808888"). Have you tried to clean the cart pins really hard ? That is the most common issue with old systems.
